common/djangoapps/track/backends/tests/test_oli.py

```diff
@@ -0,0 +1,201 @@
+"""
+Tests oli.py
+"""
+from __future__ import absolute_import
+
+from mock import patch
+import logging
+from django.test import TestCase
+from south.utils import datetime_utils as datetime
+from track.backends.oli import OLIAnalyticsBackend
+from student.tests.factories import UserFactory
+log = logging.getLogger(__name__)
+
+
+# pylint: disable=line-too-long
+class TestOLIBackend(TestCase):
+    """
+    Tests OLIAnalyticsBackend.  OLI should only process 'problem_check' event type.
+    Course IDs must be part of the whitelisted set
+    """
+    def setUp(self):
+        self.patcher = patch('track.backends.oli.OAuth1Session.put', mock_put)
+        self.addCleanup(self.patcher.stop)
+        self.patcher.start()
+        self.backend = OLIAnalyticsBackend()
+
+        # create a test user
+        self.user = UserFactory.create()
+        self.user.is_active = False
+        self.user.save()
+
+        # initialize attributes for the test cases.
+        self.backend.oli_analytics_service_url = "https://somestring"
```

Comment:
Should this be `example.com` per IANA [1]?

[1] http://www.iana.org/domains/reserved